examples/NEID_RV_test.jl

```diff
@@ -0,0 +1,152 @@
+local_dir = "/home/awise/Desktop/RvSpectML"
+#local_dir = "/storage/work/afw5465/RvSpectML"
+
+using Pkg
+ Pkg.activate(joinpath(local_dir,"alex_sandbox/"))
+ using Query, DataFrames, DataStructures, CSV, JLD, Statistics, StatsBase, Plots, JLD2, FileIO, ArgParse
+ using RvSpectMLBase, EchelleInstruments, EchelleCCFs, RvSpectML, Scalpels, RvLineList
+
+ include(joinpath(local_dir,"RvLineList/inputs/param.jl"))
+
+
+
+mask1 = RvLineList.read_mask_air(joinpath(pkgdir(RvLineList),"inputs","ESPRESSO_masks","G2.espresso.mas"))
+mask2 = RvLineList.read_mask(joinpath("/home/awise/Desktop/neid_masks/clean_masks","RvLineList_allowBlends=0_overlapcutoff=2.0e-5_rejectTelluricSlope=2000.0_badLineFilter=none,_quant=90_nbin=1_DP=true_binParam=depth_n=1_VALD_output=false_VACUUM.csv"))
+ 
+linelist_names = SortedDict(Dict())
+
+push!(linelist_names, "new_mask"=>joinpath("/home/awise/Desktop/neid_masks/clean_masks","RvLineList_allowBlends=0_overlapcutoff=2.0e-5_rejectTelluricSlope=2000.0_badLineFilter=none,_quant=90_nbin=1_DP=true_binParam=depth_n=1_VALD_output=false_VACUUM.csv"))
+push!(linelist_names, "espresso"=>joinpath(pkgdir(RvLineList),"inputs","ESPRESSO_masks","G2.espresso.mas"))
+
+pipeline_plan = PipelinePlan()
+reset_all_needs!(pipeline_plan)
+
+if need_to(pipeline_plan,:read_spectra)
+   if verbose println("# Finding what data files are avaliable.")  end
+   global df_files = RvLineList.get_NEID_best_days()
+
+   if verbose println("# Reading in customized parameters from param.jl.")  end
+   include(joinpath(local_dir,"param.jl"))
```

Comment:
It seems like local_dir is being used in two ways... path to package (can use pkgdir instead) and place to look for parameter files (maybe default to pwd)?